src/main/haskell/kore/src/Kore/Parser/ParserImpl.hs

```diff
@@ -401,6 +401,25 @@ symbolOrAliasPatternRemainderParser childParser x identifier =
         <*> inParenthesesListParser childParser
         )
 
+symbolOrAliasParser
```

Comment:
Why not `headParser`?

Reply:
I used "symbolOrAlias" because that is the name of the type, but I'm not attached to the name.

Reply:
Sorry, I meant: Why not the existing `headParser` function?